enzy_htp/structure/structure.py

```diff
@@ -212,6 +291,59 @@ def has_duplicate_chain_name(self) -> bool:
             existing_c_id.append(ch.name)
         return False
     
+    def is_idx_subset(self, target_stru: Structure) -> bool:
+        """
```

Comment:
Please update the docstring so that it is up to standard with the google style guide for python with respect to docstrings